vnext/src/Libraries/AppTheme/__mocks__/AppTheme.uwp.ts

```diff
@@ -2,11 +2,20 @@
 'use strict';
```

Comment:
Spoke with @ahimberg , the shim seems to be working, so we can remove this file/folder.